dist/tools/doccheck/generic_exclude_patterns

```diff
@@ -17,3 +17,6 @@ warning: Member LED[0-9]_PIN_NUM \(macro definition\) of
 warning: Member LED[0-9]_PORT \(macro definition\) of
 warning: Member LED[0-9]_PORT_NUM \(macro definition\) of
 warning: Member LED[0-9]_TOGGLE \(macro definition\) of
+warning: Member SHT1X_PARAMS \(macro definition\) of
+warning: Member SHT1X_PARAM_[A-Z0-9_]* \(macro definition\) of
+warning: Member SHT1X_SAULINFO \(macro definition\) of
```

Comment:
```suggestion
drivers/sht1x/include/sht1x_params\.h:[0-9]+: warning: Member SHT1X_PARAMS \(macro definition\) of
drivers/sht1x/include/sht1x_params\.h:[0-9]+: warning: Member SHT1X_PARAM_[A-Z0-9_]* \(macro definition\) of
drivers/sht1x/include/sht1x_params\.h:[0-9]+: warning: Member SHT1X_SAULINFO \(macro definition\) of
```
and put them back into exclude patterns

Reply:
@maribu added the missing generic documentation and just prepares this for some board in future having these defined (the warning does not happen)
```suggestion
```